ui/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {addLocaleData, IntlProvider} from 'react-intl';
+
+import App from './App.js';
+
+addLocaleData(require('react-intl/locale-data/' + BUNDLE_LANGUAGE));
```

Comment:
Optional: You can write
```js 
    `react-intl/locale-data/${BUNDLE_LANGUAGE}`
```
here too which may look more consistent with one below.

Reply:
Done.